### DISCUSSION TITLE
HTML Parsing Support for Script logs

### DISCUSSION THREAD

**Discussion by HeavenlyEntity** [burkybang/SuiteAdvanced-Feedback#87] [Q&A] [answered]
Hey @burkybang - here's an opportunity you can use to implement. There's another [extension ](https://chrome.google.com/webstore/detail/netsuite-html-script-note/lipldhgjkmfhamocfcdijcdgjcikcbkk) which allows for log details to be parsed when detected as **html** to format the text. Think you could replicate? I would put this apart of the config window.

**Comment by burkybang** [comment, marked as answer]
SuiteAdvanced does this already

**Reply by burkybang** [reply]
Duplicate of #44

**Reply by eduardRsult** [reply]
The part that us missing is the "Links to records." feature of that other plugin

**Reply by HeavenlyEntity** [reply]
@burkybang It doesn't with me as it explicitly gives me the literal code formatted like it would json of the html and not the html as a parsed document/formatted text log. I do have the other extension disabled.

**Reply by eduardRsult** [reply]
I do not quite understand your comment.

**Reply by burkybang** [reply]
@HeavenlyEntity I also do not understand your comment

**Reply by HeavenlyEntity** [reply]
@burkybang I get raw html text and not a parsed html formatted text in the cell.

**Reply by burkybang** [reply]
@HeavenlyEntity  Do you have any screenshot examples?

**Reply by eduardRsult** [reply]
@HeavenlyEntity do you get the same as me in this reported bug?
https://github.com/burkybang/SuiteAdvanced-Feedback/issues/91

**Reply by HeavenlyEntity** [reply]
@burkybang Here is an example of what I am referring to:

![image](https://github.com/burkybang/SuiteAdvanced-Feedback/assets/39228141/55970d4d-04fe-4d51-a40e-11e6adc42058)

**Reply by burkybang** [reply]
@HeavenlyEntity What is your expected result? What you see in the screenshot is how I would expect it to display. Are you wanting your browser to render the HTML?

**Reply by HeavenlyEntity** [reply]
@burkybang Yeah render as a parsed document like the other chrome extension. Yet make this a preference of course.

**Reply by burkybang** [reply]
@HeavenlyEntity I don't see how the other extension is rendering the HTML. Instead, it looks like it is displaying the HTML code.

**Reply by HeavenlyEntity** [reply]
@burkybang I have it rendering in my logs. The field is parsed and renders it in the container of the details. I have this working in a few install scripts. - It parses it as rich text. I can provide an example of that as well if you like?
